pkg/flavor/flavor.go

```diff
@@ -36,6 +34,8 @@ type methods struct {
 	clusterFlavorStore cache.Store
 }
 
+var _ Methods = &methods{}
```

Comment:
I think this deserves an explanation/note in the commit message.

Reply:
This is a golang idiom. It is a dummy variable, so the compiler makes sure that `*methods` struct implements `Methods` interface. It is helpful when refactoring interfaces, so we don't forget to update the structs that implement them.